google/cloud/aiplatform/metadata/metadata_store.py

```diff
@@ -242,3 +242,43 @@ def _get(
             )
         except exceptions.NotFound:
             logging.info(f"MetadataStore {metadata_store_name} not found.")
+
+    @classmethod
+    def ensure_default_metadata_store_exists(
```

Comment:
It's not directly in this PR, but what's the rationale for making this class non-public?

There's a public metadata client for it. Are there no scenarios where a user might want to manage a metadata store via the SDK?

Reply:
This interface wasn't proposed during design and needs additional considerations and review. The Metadata team is working in parallel to enable additional Metadata CUJs in the SDK.